test_project/test_project/urls.py

```diff
@@ -0,0 +1,20 @@
+"""test_project URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/1.8/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  url(r'^$', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  url(r'^$', Home.as_view(), name='home')
+Including another URLconf
+    1. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
+"""
```

Comment:
Same here - we don't need this autogenerated comment, and there's a copiable version in `base-library` should you need it.